profiles/arch/hppa/use.mask

```diff
@@ -154,6 +154,7 @@ gconf
 gnome
 gnome-keyring
```

Comment:
Do some consumers remain?

Reply:
2 remain: `mate-base/mate-session-manager`, `sys-auth/pambase`, directly pulling in `gnome-base/gnome-keyring`.

Reply:
Thanks!